### PR TITLE
chore: orchestrate CLI in monorepo with lerna

### PR DESCRIPTION
**Pull request by ryanricard**
Quick first pass. There is definite opportunity to unify repo npm scripts (e.g. linting).

### CODE REVIEW

File: lerna.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+  "lerna": "2.11.0",
+  "packages": [
+    ".",
```
Comment (AndersDJohnson) [inline lerna.json]: I hope it's okay to have a package at the root. Either way, later we may wish to refactor by moving the core module under the `packages` directory.

File: packages/cli/README.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
+# mockyeah-cli
```
Comment (AndersDJohnson) [inline packages/cli/README.md]: I think this folder should be called `mockyeah-cli` rather than just `cli` so it matches the package name on npm.

Comment (ryanricard) [reply packages/cli/README.md]: I agree.

**Comment by ryanricard** [comment]
It is my understanding that best practice would be to make linting package specific, and execute all via ‘lerna run lint’. It appears I deleted the cli dot files, they should be re-added. Plenty of opportunity to re-tool linting in a follow up PR.

File: lerna.json (hunk 1 above)
Comment (ryanricard) [reply lerna.json]: Me too. Definitely might find ourselves fighting Lerna on this one.

**Comment by AndersDJohnson** [comment]
@ryanricard Will you be taking up the work, or would you like me to help?

**Comment by ryanricard** [comment]
@AndersDJohnson, I won't get to this until tomorrow at the earliest. If you can act sooner, feel free.

**Comment by AndersDJohnson** [comment]
@ryanricard I've made some changes discussed:
* moving `packages/cli` to `packages/mockyeah-cli`
* restoring `.eslint*` files under `packages/mockyeah-cli`
* adding `packages/mockyeah-cli` to `.eslintignore` so CI passes

Would you like to merge this now as-is as an initial pass, or bite the bullet and move core `mockyeah` into `packages/mockyeah`? I don't take to touch your `git blame` if you'd rather have the credit.

**Pull request merged**